pkg/sql/create_table.go

```diff
@@ -1473,7 +1473,7 @@ func NewTableDesc(
 	}
 
 	// Create the TTL column if one does not already exist.
```

Comment:
nit: update column to `TTL automatic column (or whatever we're calling it) if one does not already `